packages/api-explorer/src/components/JsonViewer/index.jsx

```diff
@@ -6,6 +6,7 @@ import ReactJson from 'react-json-view'
 import colors from '../../colors'
 
 export default function JsonViewer({ schema, missingMessage }) {
+  const message = missingMessage ? <FormattedMessage id={missingMessage} defaultValue={'missing message'} /> : null
```

Comment:
Improve the default value. `missing message` is quite bad, it's better a missing schema default what do you think?

Reply:
yeah, is better